UI.py

```diff
@@ -213,6 +213,38 @@ async def UpdateLeaveEmbed(member, message, embed, newStatus):
     await UpdateEmbedLeaveStatus(message, embed, newStatus)
     await UpdateEmbedApprovedRejectedby(message, embed, member)
 
+def CreateCreditLeavesOptions():
+    ApplicationCommandOptionType_FLOAT = 10
+    extra_balance_options = [
+        create_option(
+            name = "discorduser",
+            description = "discord user",
+            option_type = SlashCommandOptionType.USER,
+            required = True
+        ),
+        create_option(
+            name = "leavetype",
+            description = "leave type",
+            option_type = SlashCommandOptionType.STRING,
+            required = True,
+            choices = CreateLeaveTypeChoices()
+        ),
+        create_option(
+            name = "dayscount",
+            description = "amount of extra credit (can be negative number)",
```

Comment:
Probably important to note that it defaults to 1 here

Reply:
Done